development/docker-compose.base.yml

```diff
@@ -25,8 +25,27 @@ services:
     <<: *nautobot-base
   worker:
     entrypoint: "nautobot-server rqworker"
+    healthcheck:
+      interval: "5s"
+      timeout: "5s"
+      start_period: "5s"
+      retries: 3
+      test: ["CMD", "nautobot-server", "health_check"]
     depends_on:
       - "nautobot"
+    <<: *nautobot-base
+  celery_worker:
     healthcheck:
-      disable: true
+      interval: "5s"
+      timeout: "5s"
+      start_period: "5s"
+      retries: 3
+      test: ["CMD", "nautobot-server", "health_check"]
```

Comment:
see https://github.com/nautobot/nautobot/issues/1106 and https://github.com/nautobot/nautobot/pull/1108 - it appears we shouldn't be using `nautobot-server health_check` for the celery worker.